addons/arsenal/missions/Arsenal.VR/script_component.hpp

```diff
@@ -1,4 +1,4 @@
 #include "\z\ace\addons\arsenal\script_component.hpp"
 
-#undef PREP // make.py can't redefine already defined macros
+#undef PREP
```

Comment:
Why delete this comment?

Reply:
Ah I was building with armake2 and this breaks it. I just used `commit -a` so it got tracked.

Reply:
Then move the comment above the line. That should work.

Reply:
What commy2 said, I'll get that branch merged today.

Reply:
Can be reverted!